system/kube-monitoring/charts/blackbox-exporter/templates/configmap.yaml

```diff
@@ -9,6 +9,20 @@ metadata:
 data:
   blackbox.yml: |
     modules:
+      http_manila_ca:
+        prober: http
+        http:
+          method: GET
+          preferred_ip_protocol: ip4
+          tls_config:
+            ca_file: "/SAPNetCA_G2.crt"
```

Comment:
please make configurable via values

Reply:
like this? or do we need to add wget to retrieve the cert from the main site to prevent cert expiry issues.

https://github.com/sapcc/helm-charts/blob/master/system/kube-monitoring/charts/prometheus-collector/templates/_prometheus.yaml.tpl#L101